media/CircleGraph/index.js

```diff
@@ -167,6 +170,16 @@ host.BrowserHost = class {
         case "visq":
           this._msgVisq(message);
           break;
+        case "changemode":
+          this._mode = viewMode.viewer;
+          if (message.mode === "selector") {
+            this._mode = viewMode.selector;
+          } else if (message.mode === "visq") {
+            this._mode = viewMode.visq;
+          } else if (message.mode === "visqselr") {
+            this._mode = viewMode.visqselr;
```

Comment:
only put this line as there is no other change

Reply:
> only put this line as there is no other change

Sorry. I don't quite get it. Current implementation assumes that we can change `html` `_mode` to another mode, so these lines are intended to provide full enumeration of `viewMode`, but for now only `visqselr` is enough.

Reply:
> but for now only visqselr is enough.

this is the point. check only for `visqselr`. setting to other modes are never called.